### ISSUE TITLE
Module: bigip_sslo_config_policy - No default "All Traffic" config option

### ISSUE THREAD

**Issue by kevingstewart** [F5Networks/f5-ansible-bigip#23]
##### COMPONENT NAME
Ansible module: bigip_sslo_config_policy

##### ANSIBLE VERSION
```
ansible [core 2.12.4]
  config file = /sslo/ansible/ansible.cfg
  configured module search path = ['/sslo/ansible/library']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  ansible collection location = /sslo/ansible/collection
  executable location = /usr/local/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.1
  libyaml = True
```

##### BIGIP VERSION
```
Sys::Version
Main Package
  Product     BIG-IP
  Version     15.1.1
  Build       0.0.6
  Edition     Point Release 0
  Date        Thu Oct  8 02:52:59 PDT 2020
```

##### CONFIGURATION
ansible.cfg:
```
[defaults]
host_key_checking = False
retry_files_enabled = False
inventory           = ./inventory/hosts
library             = ./library
roles_path          = ./roles
collections_paths   = ./collection
```

##### OS / ENVIRONMENT
Running Ansible inside Ubuntu:20.04 Docker container
Dockerfile:
```
FROM ubuntu:20.04

# Install components
RUN apt-get update && apt-get -y upgrade \
    && DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" apt-get install -y tzdata \
    && ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime \
    && dpkg-reconfigure --frontend noninteractive tzdata \
    && apt-get install -y gnupg software-properties-common curl awscli git python3-pip \
    && pip3 install ansible f5-sdk bigsuds netaddr objectpath isoparser lxml deepdiff \
    && curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - \
    && apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
    && apt-get update && apt-get install -y terraform

# Configure Ansible
SHELL ["/bin/bash", "-c"]
RUN mkdir -p /sslo/ansible && cd /sslo/ansible \
    && mkdir -p inventory/{group_vars,host_vars} \
    && mkdir -p {library/modules,playbooks,files,roles,scripts,templates} \
    && touch {ansible.cfg,inventory/group_vars/all.yaml,inventory/host_vars/host1.yaml,playbooks/site.yaml,inventory/hosts} \
    && echo $'[defaults]\nhost_key_checking	= False\nretry_files_enabled = False\ninventory           = ./inventory/hosts\nlibrary             = ./library\nroles_path          = ./roles\ncollections_paths   = ./collection\n' > ansible.cfg \
    && echo $'[all]\nlocalhost' > inventory/hosts \
    && ansible-galaxy collection install f5networks.f5_bigip

WORKDIR /sslo
```

##### SUMMARY
There are no configuration options for what to do with non-matching traffic in the SSLO security policy. This is the built-in "All Traffic" rule that provides actions for allow/block, intercept/bypass, and service chain selection.

##### STEPS TO REPRODUCE
No options in the policy module to create/re-create this behavior.

**Comment by trinaths** [comment]
@kevingstewart  - Please share the playbook you are using to retry at our end. This helps to understand the issue correctly.

**Comment by wojtek0806** [comment]
this one needs reopening closed by mistake

**Comment by wojtek0806** [comment]
fixed in 1.8 release
